draft-irtf-cfrg-opaque.md

```diff
@@ -412,7 +426,7 @@ This specification uses a KDF with the following API and parameters:
   into `L` bytes of output keying material.
 - Nx: The output size of the `Extract()` function in bytes.
 
-This specification also makes use of a collision-resistant Message Authentication Code
+This specification also makes use of a random-key robust Message Authentication Code
```

Comment:
I added the clause, ", defined in {{notable-design-differences}}," to help clarify it. It is a bit weird pointing to a section on notable design differences, but that is indeed where it is now defined.

We could pull it out to its own subsection within Security Considerations, i.e. 

```
## Random-Key Robustness

The random-key robustness property that we rely on for a MAC is as follows: given two random keys k1 and k2, it is infeasible to find a message m such that MAC(k1, m) = MAC(k2, m).
```

it would be a short subsection...

Reply:
Could we just inline the definition here instead?